src/Orchard.Web/Modules/Orchard.ContentTypes/Views/Admin/EditPlacement.cshtml

```diff
@@ -24,11 +24,13 @@
             
             <li data-shape-type="@placement.ShapeType" data-shape-differentiator="@placement.Differentiator" data-shape-zone="Content" data-shape-position="@placement.Position">
                 <div class="shape-type"><h3>@placement.ShapeType @placement.Differentiator</h3></div>
-                @try {
-                    <div class="shape-editor">@Display(Model.AllPlacements[i].Shape)</div>
-                }
-                catch {
-                }
+				<div class="shape-editor">
+					@try {
+						@Display(Model.AllPlacements[i].Shape)
+					}
+					catch {
+					}
+				</div>

```

Comment:
Right, so I need to replace the tab to four spaces? Sorry, it's my first PR. Also, is there somewhere I can see these "standards"? The docs has a very brief section on C# that doesn't mention spaces from what I can see.